doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1840,7 +1844,9 @@ Response:
               "attrsFormat": "keyValues",
               "attrs": ["temperature", "humidity"],
               "timesSent": 12,
-              "lastNotification": "2015-10-05T16:00:00.00Z"
+              "lastNotification": "2015-10-05T16:00:00.00Z",
+              "timesFailed": 4
```

Comment:
the `status` field will has `fail` value when the subscription is currently in failed status?  So, it value should be append in the apiary in possible values of subcription status.

**status**: `Either active (for active subscriptions) or inactive (for inactive subscriptions). If this field is not provided at subscription creation time, new subscriptions are created with the active status, which can be changed by clients afterwards. For expired subscriptions, this attribute is set to expired (no matter if the client updates it to active/inactive).`